hs628_tools/Project1_Vizualization.py

```diff
@@ -0,0 +1,297 @@
+
+# coding: utf-8
+
+# In[20]:
+
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+import pandas as pd
+import sys 
+import plotly.plotly as py
+import plotly.graph_objs as go
+#plotly.tools.set_credentials_file(username='hhowell', api_key='BRP1aHhfTHwkqnvalVQL')
+from sklearn import datasets, linear_model
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import mean_squared_error,mean_absolute_error, r2_score
+###This is importing a class from Issue #11 Test module. We used the ratio function to determine the test size ratio for the train_test_split function. 
+#from Test import data_split as ds
+
+
+###Bar Plot-Count
+#This function counts all instances in a variable and creates a barplot to display the results 
+#@param: The variable to be counted, the x_axis label, title of the graph
+#@return: A count bar plot of the variable inputted 
+def count_and_print(variable, x_label, title):
+    def count(variable):
```

Comment:
Why do you have a function inside a function. I believe it is a good practice to call a function from another function but not define the function altogether. Is there are good reason which I am missing ?

Reply:
There is not specific reason. I used this function within this function itself and only this function therefore I kept it in there. Not sure if this is a thing or not however I felt it was not a problem. Let me know!